andela/troupon
# #128054501 Edit Profile Link

#### What does this PR do?

This PR changes all the "My Profile" links to display "Edit Profile". It also adds a new "View Profile"page to display user profile details.
#### Description of Task to be completed?

Change all instances of links reading "My Profile" to display "Edit Profile" instead. Write views, urls, and template to display user profile details.
#### How should this be manually tested?

While on the homepage or the Deals page, click on the user icon in the top right corner. The first link in the dropdown should read "Profile". Clicking on it should take you to the View Profile page, where the sidebar link should also read "View Profile". The breadcrumb information should also read "View Profile".  On the sidebar of the page should be a link called "Edit Profile" that should take you to the "Edit Profile"page.
#### What are the relevant pivotal tracker stories?
#128054501
#### Screenshots

<img width="1220" alt="screen shot 2016-08-31 at 15 44 31" src="https://cloud.githubusercontent.com/assets/18420962/18129128/f08a9a7e-6f91-11e6-9089-688d8535c477.png">
<img width="692" alt="screen shot 2016-08-23 at 12 14 32" src="https://cloud.githubusercontent.com/assets/18420962/17886631/65d0b182-692b-11e6-94dc-bcf2482268f4.png">
